manifests/client/install.pp

```diff
@@ -5,8 +5,11 @@
   }
 
   if $mcollective::manage_packages {
-    package { $mcollective::client_package:
-      ensure => $mcollective::version,
+    # prevent conflict where client package name == server package name
+    if !defined(Package[$mcollective::client_package]) {
```

Comment:
I think it would be cleaner to 
```puppet
# Some systems (OpenBSD) have the same package for both
if $mco::client_package != $mco::server_package { 
  #stuff 
}
```